Lib/test/test_import/__init__.py

```diff
@@ -25,7 +25,7 @@
 
 from test.support import os_helper
 from test.support import (
-    STDLIB_DIR, swap_attr, swap_item, cpython_only, is_emscripten,
+    STDLIB_DIR, TEST_HOME_DIR, swap_attr, swap_item, cpython_only, is_emscripten,
```

Comment:
What's this import for? None of the new code seems to use it.

Reply:
It is a left-over from the old version of this PR. Thanks!